src/ResourceManager/DataMigration/Commands.DataMigration.Test/ScenarioTests/ServiceTests.ps1

```diff
@@ -312,4 +309,179 @@ function Test-MigrateSqlSqlDB
+	}
+	finally
+	{
+		Remove-ResourceGroupForTest $rg
+	}
+}
+
+function Test-MigrateSqlSqlDBMi
+{
+	$rg = Create-ResourceGroupForTest
+	
+	try
+	{
+		$service = Create-DataMigrationService($rg)
+		$project = Create-ProjectSqlSqlDbMi $rg $service
+		$taskName = Get-TaskName
+
+		#Source Connection Details
+		$sourceConnectionInfo = New-SourceSqlConnectionInfo
+		$sourceUserName = [Microsoft.Azure.Commands.DataMigrationConfig]::GetConfigString("SQL_USERNAME")
+		$sourcePassword = [Microsoft.Azure.Commands.DataMigrationConfig]::GetConfigString("SQL_PASSWORD")
+		$sourceCred = Get-Creds $sourceUserName $sourcePassword
+
+		#Target Connection Details
+		$targetConnectionInfo = New-TargetSqlMiConnectionInfo
+		$targetUserName = [Microsoft.Azure.Commands.DataMigrationConfig]::GetConfigString("SQLDBMI_USERNAME")
+		$targetPassword = [Microsoft.Azure.Commands.DataMigrationConfig]::GetConfigString("SQLDBMI_PASSWORD")
+		$targetCred = Get-Creds $targetUserName $targetPassword
+		
+		$blobSasUri = [Microsoft.Azure.Commands.DataMigrationConfig]::GetConfigString("BLOB_SAS_URI")
+		$fileSharePath = [Microsoft.Azure.Commands.DataMigrationConfig]::GetConfigString("FILESHARE_PATH")
+		$fileShareUsername = [Microsoft.Azure.Commands.DataMigrationConfig]::GetConfigString("FILESHARE_USERNAME")
+		$fileSharePassword = [Microsoft.Azure.Commands.DataMigrationConfig]::GetConfigString("FILESHARE_PASSWORD")
+		$fileShareCred = Get-Creds $fileShareUsername $fileSharePassword
+
+		$backupFileShare = New-AzureRmDmsFileShare -Path $fileSharePath -Credential $fileShareCred
+
+		$selectedDbs = New-AzureRmDataMigrationSelectedDB -MigrateSqlServerSqlDbMi -Name "HR" -TargetDatabaseName "HR_PSTEST" -BackupFileShare $backupFileShare
+
+        $selectedLogins = [Microsoft.Azure.Commands.DataMigrationConfig]::GetConfigString("MI_LOGINS")
+        $selectedLogins = $selectedLogins -split ","
+
+        $selectedJobs = [Microsoft.Azure.Commands.DataMigrationConfig]::GetConfigString("MI_AGENTJOBS")
+        $selectedJobs = $selectedJobs -split ","
+
+		$migTask = New-AzureRmDataMigrationTask -TaskType MigrateSqlServerSqlDbMi `
+		  -ResourceGroupName $rg.ResourceGroupName `
+		  -ServiceName $service.Name `
+		  -ProjectName $project.Name `
+		  -TaskName $taskName `
+		  -TargetConnection $targetConnectionInfo `
+		  -TargetCred $targetCred `
+		  -SourceConnection $sourceConnectionInfo `
+		  -SourceCred $sourceCred `
+		  -BackupBlobSasUri $blobSasUri `
+		  -BackupFileShare $backupFileShare `
+		  -SelectedDatabase $selectedDbs `
+          -SelectedAgentJobs $selectedJobs `
+          -SelectedLogins $selectedLogins 
+
+		$task = Get-AzureRmDataMigrationTask -ResourceGroupName $rg.ResourceGroupName -ServiceName $service.Name -ProjectName $project.Name -TaskName $taskName -Expand
+
+		Assert-AreEqual $taskName $task[0].Name
+		Assert-AreEqual 1 $task.Count
```

Comment:
Are there any more properties you can check to make sure this was set up properly?

Reply:
Fixed.